src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,512 @@
+| Units       | REQUIRED. Unit of radioactivity measurements in whole blood samples (for example, "kBq/mL") |
+
+### Example (`*_blood.json`)
+
+```JSON
+{
+	"PlasmaAvail": true,
+	"MetaboliteAvail": true,
+	"ContinuousBloodAvail": true,
+	"DiscreteBloodAvail": true
+}
+```
+
+### Example (`*_recording-blood_discrete.json`)
+
+```JSON
+{
+    "time": {
+        "Description": "Time in relation to time zero defined by the _pet.json",
+        "Units": "s"
+    },
+    "plasma_radioactivity": {
+        "Description": "Radioactivity in plasma samples. Measured using COBRA counter.",
+        "Units": "kBq/mL"
+    },
+    "whole_blood_radioactivity": {
+        "Description": "Radioactivity in whole blood samples. Measured using COBRA counter.",
+        "Units": "kBq/mL"
+    },
+    "metabolite_parent_fraction": {
+        "Description": "Parent fraction of the radiotracer.",
+        "Units": "unitless"
+    },
+    "metabolite_polar_fraction": {
+        "Description": "Polar metabolite fraction of the radiotracer.",
+        "Units": "unitless"
+    },
+    "metabolite_lipophilic_fraction": {
+        "Description": "Lipophilic metabolite fraction of the radiotracer.",
+        "Units": "unitless"
+    }
+}
+```
+
+![Figure 2](images/blood_discrete_tsv.png "Screenshot of the discrete blood tsv file.")
+
+Figure 2: Caption of the corresponding `recording-blood_discrete.tsv` file.
+
+
+### Example (`*_recording-blood_continuous.json`)
+
+```JSON
+{
+"time": {
+        "Description": "Time in relation to time zero defined by the _pet.json",
+        "Units": "s"
+    },
+    "plasma_radioactivity": {
+        "Description": "Radioactivity in plasma samples. Measured using Allogg autosampler.",
+        "Units": "kBq/mL"
+    }
+}
+```
+
+# Appendix: Useful resources
+
+This website is an extremely useful resource: http://www.turkupetcentre.net/petanalysis/quantification.html
+
+An extensive resource for background knowledge on kinetic modeling
+can be found here: https://nru.dk/index.php/misc/category/3-pet-kinetic-course
+
+# References
+
+[1] Knudsen, G. M., Ganz, M., Appelhoff, S., Boellaard, R., Bormans, G., Carson, R. E., … Innis, R.B. (2020). Guidelines for the content and format of PET brain data in publications and archives : A consensus paper. Journal of Cerebral Blood Flow and Metabolism. https://doi.org/10.1177/0271678X20905433
+
+[2] Nørgaard, M., Ganz, M., Svarer, C., Frokjaer, V. G., Greve, D. N., Strother, S. C., & Knudsen, G. M. (2019). Optimization of preprocessing strategies in Positron Emission Tomography (PET) neuroimaging: A [11C]DASB PET study. NeuroImage, 199(October 2019), 466–479. https://doi.org/10.1016/j.neuroimage.2019.05.055
+
+[3] Innis, R. B., Cunningham, V. J., Delforge, J., Fujita, M., Gjedde, A., Gunn, R. N., … Carson, R. (2007). Consensus nomenclature for in vivo imaging of reversibly binding radioligands. Journal of Cerebral Blood Flow and Metabolism, 1–7. https://doi.org/10.1038/sj.jcbfm.9600493
```

Comment:
I suggest to not list this "references" section, and instead edit the in-text references to contain DOI links to the papers.

For example up there in the text this:

`(Knudsen et al. 2020, [1])`

would become this:

`(Knudsen et al. 2020, [doi:10.1177/0271678X20905433](https://doi.org/10.1177/0271678X20905433))`

which is rendered like this:

> (Knudsen et al. 2020, [doi:10.1177/0271678X20905433](https://doi.org/10.1177/0271678X20905433))

We may expand the BIDS specification to include such "reference" sections in the future (see the request here: https://github.com/bids-standard/bids-specification/issues/403) --> but currently we do not have it that way (to my knowledge).

Reply:
Hi @sappelhoff, has this changed since then? So which way would you like us to do t?

Reply:
My recommendation from Oct 21 still stands :-) there have been no changes to how we handle DOIs meanwhile.